src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -74,14 +77,26 @@ const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = (props) => {
             </div>
           </div>
         </div>
-        <div className="detail-bottom-wrapper">
-          <div className="detail-select-studies">
-            {studyInstances.map((study: StudyInstance, i) => (
-              <SelectionStudy key={i} {...study}></SelectionStudy>
-            ))}
-          </div>
-          <SelectedStudyDetail></SelectedStudyDetail>
-        </div>
+        <PageSection className="PatientLookupWrapper" variant={PageSectionVariants.light}>
+          <FileLookup />
+        </PageSection>
+        {
+          studyInstances.length > 0 ?
+            <div className="detail-bottom-wrapper">
+              <div className="detail-select-studies">
+                {studyInstances.map((study: StudyInstance, i) => (
+                  <SelectionStudy key={i} {...study}></SelectionStudy>
```

Comment:
Using `array index` as a `key` prop is an **anti pattern**. Please use a unique key.